Composer/packages/server/src/controllers/publisher.ts

```diff
@@ -240,4 +240,50 @@ export const PublishController = {
       message: `${method} is not a valid publishing target type. There may be a missing plugin.`,
     });
   },
+  removeLocalRuntimeData: async (req, res) => {
```

Comment:
I don't think this functionality should be exposed as an external API. This is very specific to one type of publishing scenario -- the local runtime.

I would propose that the the "delete bot" feature simply call the stopbot and removeLocalRuntimeData directly.